pages/index.tsx

```
@@ -49,6 +67,83 @@ export default function Home() {
+      case 'tone':
+        if (!value.trim()) return 'Tone helps guide the story mood';
+        return undefined;
+      case 'setting':
+        if (!value.trim()) return 'Setting provides important context';
+        return undefined;
+      case 'visualVibe':
+        if (!value.trim()) return 'Visual style reference helps generate better art';
+        return undefined;
+      case 'castInput':
+        const characters = value.split('\n').filter(s => s.trim());
+        if (characters.length === 0) return 'At least one character is required';
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)